### PR TITLE
Merge generic and Core controls doc pages

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Addresses #558.

+ Add breadscumbs in element doc pages

### CODE REVIEW

File: docs/manuals/gui/viselements/controls.md_template [hunk 1]
```diff
@@ -1,10 +1,24 @@
 # Controls
 
-Controls are user interface objects that represent data.
+Controls are user interface objects that represent data and that users can interact with.
 
-## Controls list
+## Generic controls
 
-Here is the list of all available controls in Taipy:
+Here is the list of all available controls in Taipy GUI:
```
Comment (FredLL-Avaiga) [inline docs/manuals/gui/viselements/controls.md_template]: Gui is still in?

**Review by FredLL-Avaiga — APPROVED** [review]

File: docs/manuals/gui/viselements/controls.md_template [hunk 2]
```diff
@@ -1,10 +1,24 @@
 # Controls
 
-Controls are user interface objects that represent data.
+Controls are user interface objects that represent data and that users can interact with.
 
-## Controls list
+## Generic controls
```
Comment (jrobinAV) [inline docs/manuals/gui/viselements/controls.md_template]:
```suggestion
## GUI controls
```

File: docs/manuals/gui/viselements/controls.md_template (hunk 1 above)
Comment (jrobinAV) [reply docs/manuals/gui/viselements/controls.md_template]: What about removing GUI here and changing the section title to "Gui controls"?

**Review by jrobinAV — APPROVED** [review]

File: docs/manuals/gui/viselements/controls.md_template (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/controls.md_template]: "Gui controls" to me sounds very awkward. There can be no controls without a GUI.
The term "Generic" indicates that these are available no matter what. That is, having taipy-gui installed.
Let's discuss this further. I'm not really happy either.

File: docs/manuals/gui/viselements/controls.md_template (hunk 2 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/controls.md_template]: See above.

File: docs/manuals/gui/viselements/controls.md_template (hunk 1 above)
Comment (FabienLelaquais) [reply docs/manuals/gui/viselements/controls.md_template]: I rephrased that a little bit, using the "Standard controls" term for the ones that specifically are in taipy-gui.
Also added a note that explains that (as soon as taipy-gui is installed, you're good to go).

**Pull request merged**